UnitTests/BraintreeTestShared/MockAPIClient.swift

```diff
@@ -17,6 +17,7 @@ public class MockAPIClient: BTAPIClient {
     public var postedIsVaultRequest = false
     public var postedMerchantExperiment: String? = nil
     public var postedPaymentMethodsDisplayed: String? = nil
+    public var postedAppSwitchURL: [String: URL?] = [:]
```

Comment:
I think this would just be a string of the URL, right?

Reply:
You're right, updated: [4712ec0](https://github.com/braintree/braintree_ios/pull/1435/commits/4712ec0f96259bd101dc9c8a8713c3860d96012c)